gm4_metallurgy_moneo/pack.mcmeta

```diff
@@ -0,0 +1,25 @@
+{
+  "pack":{
+    "pack_format":0,
+    "description":"A Gamemode 4 Module"
+  },
+  "module_name":"Metallurgy Moneo",
+  "module_id":"metallurgy_moneo",
```

Comment:
This should be `Moneo Shamir` and `moneo_shamir` imo (also applies in `init.mcfunction`), but feel free to discuss this as there haven't been any shamir expansions yet.

The translation key should then also be `module.gm4.moneo_shamir`

Reply:
Alright that seems reasonable, I'll make the requested changes.

Reply:
What about the installation text? It currently says `Metallurgy Expansion (Moneo)`. I'm assuming that should just be `Moneo Shamir`?

Reply:
Yup! It's basically a standalone module feature wise, just using metallurgy for crafting.